autopush/endpoint.py

```diff
@@ -17,73 +17,160 @@
 If no external routers are configured then the :class:`RegistrationHandler`
 will not be able to perform any additional router data registration.
 
-HTTP API
-========
+Push Service HTTP API
+=====================
+
+The following section describes how remote servers can send Push
+Notifications to apps running on remote User Agents.
 
 API methods requiring Authorization must provide the Authorization
 header containing the authrorization token. The Authorization token is returned
 as "secret" in the registration response.
 
-All message bodies must be UTF-8 encoded.
+Lexicon
+-------
+
+   :{UAID}: The Push User Agent Registration ID
+
+Push assigns each remote recipient a unique identifier. This value is
+assigned during **Registration**
+
+   :{CHID}: The Channel Subscription ID
+
+Push assigns each channel subscription for a given {UAID} a unique
```

Comment:
Copyedit: "assigns a unique identifier to each subscription for a given {UAID}."